### PR TITLE
Update perf_tuning.md

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

File: doc/manuals/admin/perf_tuning.md [hunk 1]
```diff
@@ -40,19 +40,28 @@ speeds up reads) and write efficiency (the usage of indexes slows down writes) a
...
 
-However, in order to help administrators in this task, the following indexes are recommended:
+However, in order to help administrators in this task, the following *general* indexes are recommended:
 
 * Collection [entities](database_model.md#entities-collection)
-    * `_id.id`
-    * `_id.type`
-    * `_id.servicePath`
-    * `attrNames`
+    * `{_id.servicePath: 1, _id.id: 1, _id.type: 1}` (note this is a compound index and key order matters in this case)
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: note:
      OR
note that ...

File: doc/manuals/admin/perf_tuning.md [hunk 2]
```diff
@@ -40,19 +40,28 @@ speeds up reads) and write efficiency (the usage of indexes slows down writes) a
...
     * `creDate`
 
-In the case of using `orderBy` queries (i.e. `GET /v2/entities?orderBy=A`), it is also recommended 
+In addition, depending on your queries you could need *additional* indexes.
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: could => may
Also, I'd add a comma after queries:

In addition, depending on your queries, you may need *additional* indexes.

File: doc/manuals/admin/perf_tuning.md [hunk 3]
```diff
@@ -40,19 +40,28 @@ speeds up reads) and write efficiency (the usage of indexes slows down writes) a
...
+
+* In the case of using attribute filters in queries (e.g. `GET /v2/entities?q=A<10`), it is recommended to create
+indexes for them. In particular if you are filtering by a given attribute 'A' then you should create an index 
+`{attrs.A.value: 1}`. If you are filtering by several attributes in the same query (e.g. `GET /v2/entities?q=A<10;B>20`
+you should combine all them in a compound index `{attrs.A.value: 1, attrs.B.value: 1}` (key order doesn't matter in this case).
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: combine all them => combine them all    OR    combine all of them

File: doc/manuals/admin/perf_tuning.md [hunk 4]
```diff
@@ -40,19 +40,28 @@ speeds up reads) and write efficiency (the usage of indexes slows down writes) a
...
+
+* In the case of using `orderBy` queries (i.e. `GET /v2/entities?orderBy=A`), it is recommended 
 to create indexes for them. In particular, if you are ordering by a given attribute 'A' in ascending order
 (i.e. `orderBy=A`) you should create an index `{attrs.A.value: 1}`. In the case of ordering by a given
-attribute 'A' in descending order (i.e. `orderBy=!A`) you should create an index `{attrs.A.value: -1}`.
+attribute 'A' in descending order (i.e. `orderBy=!A`) you should create an index `{attrs.A.value: -1}`. In the case
+of using several attributes for ordering (i.e. `orderBy=A,!B,C`) you should create a compound index taking into
+account ordering direction, i.e. `{attrs.A.value: 1, attrs.B.value: -1, attrs.C.value: 1}` (key order matters in this case)
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: taking into￼ account **the** ordering direction

File: doc/manuals/admin/perf_tuning.md [hunk 5]
```diff
@@ -61,7 +70,7 @@ Orion startup or when entities are created.
 * An index with `expireAfterSeconds: 0` for the `expDate` field in the entities collection, due to functional needs of the [transient entities functionality](../user/transient_entities.md).
 
 You can find an analysis about the effect of indexes in [this document](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/extra/indexes_analysis.md), although
-it is based on an old Orion version, so it is probably outdated.
+it is based on an old Orion version, so it is probably outdated. In addition, please find some useful references about [compound indexes](https://docs.mongodb.com/manual/core/index-compound/) and [sort results with indexes](https://docs.mongodb.com/manual/tutorial/sort-results-with-indexes/) from MongoDB official documentation.
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: from **the** MongoDB official documentation   OR:
from **the** official documentation of MongoDB

File: doc/manuals/admin/perf_tuning.md (hunk 1 above)
Comment (fgalan) [reply doc/manuals/admin/perf_tuning.md]: Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd

File: doc/manuals/admin/perf_tuning.md (hunk 2 above)
Comment (fgalan) [reply doc/manuals/admin/perf_tuning.md]: Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd

File: doc/manuals/admin/perf_tuning.md (hunk 3 above)
Comment (fgalan) [reply doc/manuals/admin/perf_tuning.md]: Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd

File: doc/manuals/admin/perf_tuning.md (hunk 4 above)
Comment (fgalan) [reply doc/manuals/admin/perf_tuning.md]: Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd

File: doc/manuals/admin/perf_tuning.md (hunk 5 above)
Comment (fgalan) [reply doc/manuals/admin/perf_tuning.md]: Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd

File: doc/manuals/admin/perf_tuning.md (hunk 1 above)
Comment (kzangeli) [reply doc/manuals/admin/perf_tuning.md]: Sorry, I wasn't clear in my comment ...
note that this is a ...

Comment (fgalan) [reply doc/manuals/admin/perf_tuning.md]: Fixed in 6efa4d8e596c4271e7fc7ddf7fbac7aea3b92679

**Comment by kzangeli** [comment]
LGTM

**Comment by fgalan** [comment]
@fisuda could you implemented the associated changes in the Japanese translation, please?

Thanks!

**Comment by fisuda** [comment]
Noted with thanks.

**Pull request merged**